src/components/Login/Login.js

```diff
@@ -75,7 +75,7 @@ class Login extends Component {
                 <Grid item xs={12} sm={6} style={{textAlign: 'center'}}>
                     <h4>Enter your credentials to login and start shit posting</h4>
                     {this.props.user.error &&
-                        <FormHelperText error>{this.props.user.error}</FormHelperText>
+                        <FormHelperText error style={{textAlign: 'center'}}>{this.props.user.error}</FormHelperText>
```

Comment:
Probably worth putting this into `styles` and importing it from that.

Reply:
What I meant was you can do add it to:

https://github.com/DarthHater/bored-board-webapp/blob/928c02aadb6691eede00c760796f3d42dbf18deb/src/components/Login/Login.js#L27

like:

```
    santaDog: {
        textAlign: 'center'
    },
    formHelper: {
       textAlign: 'center'
    }
```

And import like `style={styles.formHelper}`

This keeps the style isolated to the class, which is kinda nice.